Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7
+FROM python:slim-buster
```

Comment:
Es buena practica colocar el numero de la versión, en este caso python3

Reply:
@Scot3004 creo que esto está bien. Se refiere a que usará la imagen python:slim-buster de Docker

Reply:
si, esta bien solo lo digo por que esa versión varia con el tiempo 
![imagen](https://user-images.githubusercontent.com/1175402/113789354-8f040200-9704-11eb-9e7d-e356bff6d729.png)
 cuando salga una nueva versión de python entonces va a cambiar esa imagen

Reply:
Yo había buscado en los tags en Dockerhub y python:slim-buster apunta a la última versión 3.x

![Captura de pantalla de 2021-04-06 18-37-42](https://user-images.githubusercontent.com/45174468/113790694-87922800-9707-11eb-914e-b9c9d530a181.png)

Reply:
Hay otro tag que es python:3-slim-buster que apunta a la misma versión 3.x

![Captura de pantalla de 2021-04-06 18-39-25](https://user-images.githubusercontent.com/45174468/113790796-d3dd6800-9707-11eb-9586-e727cf275413.png)

Reply:
Si, es correcto como esto no es crítico no hay problemas como está, esa sugerencia agregaría más valor en el caso que necesitemos estabilidad

Reply:
¿Se debe cambiar algo en el Pull Request?

Reply:
No creo, este comment es un nice to have